core-bundle/contao/dca/tl_form.php

```diff
@@ -261,6 +261,13 @@
 			'inputType'               => 'checkbox',
 			'eval'                    => array('tl_class'=>'w50'),
 			'sql'                     => array('type' => 'boolean', 'default' => false)
+		),
+		'ajax' => array
+		(
+			'filter'                  => true,
+			'inputType'               => 'checkbox',
+			'eval'                    => array('tl_class'=>'w50'),
+			'sql'                     => array('type' => 'boolean', 'default' => true)
```

Comment:
I think we should add a migration that adds this column with a default values of `false` for all existing forms, so that only newly created forms get this activated by default.

Reply:
But it only has an effect if you provided a message, does it not? Also imho it should simply not be `true` by default.

Reply:
I don't really mind. Can you guys agree on a way forward then, please? :)

Reply:
Wouldn't an SQL default value of `false` and a DCA default value of  `true` work to have BC and still the right value when creating new records? Or does this not work with booleans?

I'm fine with any version btw. 😌

Reply:
@m-vo that would technically work, but the intentions would not be clear from reading the code. That is why we use migrations in these cases AFAIK.

> Also imho it should simply not be `true` by default.

I think I’d agree. And in this case we also don’t need a migration.

```suggestion
			'sql'                     => array('type' => 'boolean', 'default' => false)
```